api/policies/v1/admissionpolicygroup_webhook_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build testing
```

Comment:
The `testing` build tag is unnecessary, as this file is already excluded by the final binary being a test.